src/systems/systems/src/pressurization/cabin_pressure_controller.rs

```diff
@@ -673,16 +719,24 @@ mod pressure_schedule_manager_tests {
 
     struct TestAircraft {
         cpc: CabinPressureController,
+        outflow_valve: PressureValve,
+        press_overhead: PressurizationOverheadPanel,
         engine_1: TestEngine,
         engine_2: TestEngine,
+        lgciu_gear_compressed: bool,
```

Comment:
**nitpick**: You could consider renaming this to `lgciu_gears_compressed`, as it's both the left and right gear that are compressed, so not a single one.

Reply:
Done 😄